src/helpers/composeVersionChecker.ts

```diff
@@ -0,0 +1,103 @@
+import compareVersions from 'compare-versions'
+
+interface IVersionMap {
+  [key: string]: string
+}
+
+// Here the engine version is the lowest needed to support all docker compose features
+const composeVersionMap: IVersionMap = {
+  '3.7': '18.06.0',
+  '3.6': '18.02.0',
+  '3.5': '17.12.0',
+  '3.4': '17.09.0',
+  '3.3': '17.06.0',
+  '3.2': '17.04.0',
+  '3.1': '1.13.1',
+  '3.0': '1.13.0',
+  '2.4': '17.12.0',
+  '2.3': '17.06.0',
+  '2.2': '1.13.0',
+  '2.1': '1.12.0',
+  '2.0': '1.10.0',
+  '1.0': '1.9.1',
+}
+
+// Here the engine version is the greatest for which the api support all features
+const apiVersionMap: IVersionMap = {
+  '1.38': '18.06',
+  '1.37': '18.05',
+  '1.36': '18.02',
+  '1.35': '17.12',
+  '1.34': '17.11',
+  '1.33': '17.10',
+  '1.32': '17.09',
+  '1.31': '17.07',
+  '1.30': '17.06',
+  '1.29': '17.05',
+  '1.28': '17.04',
+  '1.27': '17.03.1',
+  '1.26': '17.03',
+  '1.25': '1.13',
+  '1.24': '1.12',
+  '1.23': '1.11',
+  '1.22': '1.10',
+  '1.21': '1.9',
+  '1.20': '1.8',
+  '1.19': '1.7',
+  '1.18': '1.6',
+}
+
+function normalizeComposeVersion(composeVersion: string): string {
+  return composeVersion.replace(/^(\d+)$/, '$1.0')
+}
+
+function cleanEngineVersion(baseEngineVersion: string): string {
```

Comment:
Since the compose version of this function is called `normalizeComposeVersion` shouldn't this one called `normalizeEngineVersion` ?